packages/alignments/src/PileupTrack/model.ts

```diff
@@ -118,7 +118,8 @@ export default (
           ...self.composedRenderProps,
           ...getParentRenderProps(self),
           trackModel: self,
-          sortObject: self.sortObject,
+          sortObject: this.sortObject,
+          showSoftClip: getParentRenderProps(self).trackModel.showSoftClipping,
```

Comment:
at some point, the PileupTrack should really take control of this variable and not have to look to it's parent for this IMO, but I wouldn't call it a blocker here.